### PR TITLE
Added privacy overview

### PR DESCRIPTION
**Pull request by MadelineMurray**
Added privacy overview

### CODE REVIEW

File: docs/Privacy/Privacy-Overview.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+Other parties cannot access the transaction content, sending party, or list of participating parties. 
+
+## Private Transaction Manager
+
+Pantheon uses a Private Transaction Manager to implement privacy. For example, [Orion](http://docs.orion.pegasys.tech).  
+Each Pantheon node that sends or receives private transactions requires an associated Orion node. 
+
+![Orion Nodes](../images/OrionNodes.png)
+
+Private transactions are passed from the Pantheon node to the associated to the Orion node. The Orion node
```
Comment (NicolasMassart) [inline docs/Privacy/Privacy-Overview.md]:
```suggestion
Private transactions are passed from the Pantheon node to the associated Orion node. The Orion node
```

File: docs/Privacy/Privacy-Overview.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    * A, B, and C to send transactions that are private from D 
+    * A, C, and D to send transactions that are private from B 
+
+    To send private transactions between A, B, and C, A initialises a contract in a private transaction with
+    B and C specified as the `privateFor` and A specified as the `privateFrom`. Initialising the contract 
+    creates a privacy group consisting of A, B, and C. For the ABC private state to remain consistent, 
+    A, B, and C must be included on transactions (as either `privateFrom` or `privateFor`) even if they are 
+    between two of the three parties.
+
+    To send private trasactions between A, C, and D, C initialises a different contract in a private transaction with
```
Comment (NicolasMassart) [inline docs/Privacy/Privacy-Overview.md]:
```suggestion
    To send private transactions between A, C, and D, C initialises a different contract in a private transaction with
```

File: docs/Privacy/Privacy-Overview.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/Privacy/Privacy-Overview.md]: Fixed

File: docs/Privacy/Privacy-Overview.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/Privacy/Privacy-Overview.md]: Fixed

File: docs/Privacy/Privacy-Overview.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+## Private Transaction Manager
+
+Pantheon uses a Private Transaction Manager to implement privacy. For example, [Orion](http://docs.orion.pegasys.tech).  
+Each Pantheon node that sends or receives private transactions requires an associated Orion node. 
+
+![Orion Nodes](../images/OrionNodes.png)
+
+Private transactions are passed from the Pantheon node to the associated to the Orion node. The Orion node
+encrypts and directly distributes the private transaction to Orion nodes participating in the transaction.
```
[the remark this answers is not in the document]
Comment (MadelineMurray) [reply docs/Privacy/Privacy-Overview.md]: Fixed

File: docs/Privacy/Privacy-Overview.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+!!! important 
+    The mapping of Pantheon node addresses to Orion node public keys is off-chain.  That is, the 
+    sender of a private transaction must know the Orion node public key of the recipient.  
+ 
+## Privacy Groups 
+
+The group of nodes specified by `privateFrom`and `privateFor` form a privacy group and 
+are given a unique privacy group ID by Orion. The private transaction is stored in Orion with the privacy group ID. 
+
+The Pantheon nodes maintain the public world state and a private state for each privacy group. The private
```
[the remark this answers is not in the document]
Comment (MadelineMurray) [reply docs/Privacy/Privacy-Overview.md]: Good point. Fixed

**Pull request merged**